ICP.py

```diff
@@ -0,0 +1,109 @@
+import torch
+import pypose as pp
+from torch import nn
+
+
+class ICP(nn.Module):
+    r'''
+    Iterative Closest Point (ICP) using Singular Value Decomposition (SVD).
+
+    Args:
+        max_iterations: the max iteration number
+        tolerance: the tolerance of error for early stopping
+        is_matching: whether the input points set and the target points set have been matched
+    '''
+
+    def __init__(self, max_iterations=200, tolerance=0.0001, is_matching=False):
+        super().__init__()
+        self.max_iterations = max_iterations
+        self.tolerance = tolerance
+        self.is_matching = is_matching
+
+    def forward(self, p1, p2):
+        ori_p2 = p2.clone()
+        temp_pc = p1.clone()
+        iter_num = 0
+        prev_err = 0
```

Comment:
As Zitong mentioned, the usage of `_` can be avoided. The variable name p1 and p2 can be improved to make it clearer. And 'ori_p2' can be simplified as 'original', 'originalpc' or 'existpc', etc. Given we only have pointcloud here, I think it's clear enough to use "original" and "temp", but if we want to make it clearer, 'existpc' could be better. Some other improvements can be done:
iter_num -> iteration
prev_err -> error